test/e2e/clusterlogging_test.go

```diff
@@ -101,7 +101,7 @@ func clusterLoggingFullClusterTest(t *testing.T, f *framework.Framework, ctx *fr
 			Curation: logging.CurationSpec{
 				Type: "curator",
 				CuratorSpec: logging.CuratorSpec{
-					Schedule: "30 3 * * *",
+					Schedule: "* * * * *",
```

Comment:
will this value always be replaced?  If not, `"30 3 * * *"` is a good default value